tools/devops/run-tests.sh

```diff
@@ -0,0 +1,15 @@
+set -x
+set -e
+
+P=$(cat tmp.p)
+
+if grep ios-64 xamarin-macios/tests/xharness/Jenkins.cs 2>&1 > /dev/null; then
```

Comment:
You don't need the `ios-64` check here, you know that this check will always turn out true (it was needed when this code was in Azure Devops, because then it could execute for old branches, but that won' happen when the code is in git).

Reply:
Ooooooooooh you're riiiiiiiiight!